### PR TITLE
[DOCS] Merge Samples Articles Language Versions

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
This PR introduces merged Samples articles with instructions for both Python and C++.

### CODE REVIEW

File: docs/articles_en/learn_openvino/openvino_samples.rst [hunk 1]
```diff
@@ -17,96 +17,80 @@ OpenVINO™ Samples
...
+   openvino_sample_hello_classification
+   openvino_sample_hello_reshape_ssd
+   openvino_sample_hello_nv12_input_classification
+   openvino_sample_hello_query_device
+   openvino_sample_model_creation
+   openvino_sample_automatic_speech_recognition
+   openvino_sample_sync_benchmark
+   openvino_sample_throughput_benchmark
+   openvino_sample_bert_benchmark
+   openvino_sample_benchmark_tool
```
Comment (msmykx-intel) [inline docs/articles_en/learn_openvino/openvino_samples.rst:30]:
```suggestion
   openvino_sample_hello_classification
   openvino_sample_hello_nv12_input_classification
   openvino_sample_hello_query_device
   openvino_sample_hello_reshape_ssd
   openvino_sample_image_classification_async
   openvino_sample_model_creation
   openvino_sample_sync_benchmark
   openvino_sample_throughput_benchmark
   openvino_sample_bert_benchmark
   openvino_sample_benchmark_tool
   openvino_sample_automatic_speech_recognition
```

File: docs/articles_en/learn_openvino/openvino_samples.rst [hunk 2]
```diff
@@ -17,96 +17,80 @@ OpenVINO™ Samples
...
+  .. code-block:: console
 
-- **Automatic Speech Recognition Sample** - ``[DEPRECATED]`` Acoustic model inference based on Kaldi neural networks and speech feature vectors.
+     benchmark_app -m <model> -i <input> -d <device>
 
-  - :doc:`Python Sample <openvino_inference_engine_ie_bridges_python_sample_speech_sample_README>`
-  - :doc:`C++ Sample <openvino_inference_engine_samples_speech_sample_README>`
+- :doc:`Automatic Speech Recognition Sample <openvino_sample_automatic_speech_recognition>` - 
+  ``[DEPRECATED]`` Acoustic model inference based on Kaldi neural networks and 
+  speech feature vectors.
```
Comment (msmykx-intel) [inline docs/articles_en/learn_openvino/openvino_samples.rst:89]:
```suggestion
- ``[DEPRECATED]`` :doc:`Automatic Speech Recognition Sample <openvino_sample_automatic_speech_recognition>` - 
  Acoustic model inference based on Kaldi neural networks and 
  speech feature vectors.
```

File: docs/articles_en/learn_openvino/openvino_samples/automatic_speech_recognition.rst [hunk 3]
```diff
@@ -0,0 +1,656 @@
+.. {#openvino_sample_automatic_speech_recognition}
+
+Automatic Speech Recognition Sample
+===================================
```
Comment (msmykx-intel) [inline docs/articles_en/learn_openvino/openvino_samples/automatic_speech_recognition.rst:4]:
```suggestion
[DEPRECATED] Automatic Speech Recognition Sample
====================================================
```

File: docs/articles_en/learn_openvino/openvino_samples/model_creation.rst [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+| Model Format      | Model weights file (\*.bin)                                          |
++-------------------+----------------------------------------------------------------------+
+| Supported devices | :doc:`All <openvino_docs_OV_UG_supported_plugins_Supported_Devices>` |
++-------------------+----------------------------------------------------------------------+
+
+
+How It Works
+####################
+
+At startup, the sample application reads command-line parameters, `builds a model <openvino_docs_OV_UG_Model_Representation>`
```
Comment (msmykx-intel) [inline docs/articles_en/learn_openvino/openvino_samples/model_creation.rst:35]:
```suggestion
At startup, the sample application reads command-line parameters, :doc:`builds a model <openvino_docs_OV_UG_Model_Representation>` 
```

File: docs/articles_en/learn_openvino/openvino_samples/automatic_speech_recognition.rst [hunk 5]
```diff
@@ -0,0 +1,656 @@
...
+         mo --framework kaldi --input_model wsj_dnn5b.nnet --counts wsj_dnn5b.counts --remove_output_softmax --output_dir <OUTPUT_MODEL_DIR>
+
+
+The following pre-trained models are available:
+
+- rm_cnn4a_smbr
+- rm_lstm4f
+- wsj_dnn5b_smbr
+
+All of them can be downloaded from `the storage <https://storage.openvinotoolkit.org/models_contrib/speech/2021.2>`.
```
Comment (msmykx-intel) [inline docs/articles_en/learn_openvino/openvino_samples/automatic_speech_recognition.rst:351]:
```suggestion
- ``rm_cnn4a_smbr``
- ``rm_lstm4f``
- ``wsj_dnn5b_smbr``

All of them can be downloaded from `the storage <https://storage.openvinotoolkit.org/models_contrib/speech/2021.2>`__ .
```

**Comment by ilya-lavrenov** [comment]
## Review:

1. For some reason MO commands are still used, while it's a deprecated tool. Please, remove MO commands from other samples as well, we need to recommend to use OVC instead
image

2. Here, Inference Engine Plugin in C API is mentioned. Why do we need to distinguish between C and C++ API? Inference Engine is obsolete name
image

3. It's stated that sample supports IR and ONNX files, while it supports any file format supported by `Core::read_model`. Similar comments to other samples
image

4. For some reasons C++ tab provides MO command. I suppose this tabs to convert model should be Python / OVC only. Note, that `Python` for model conversion is different from `Python` which run samples. E.g. I can convert model via Python script and then use it in C++ sample. Similar comment to other samples
image

5. Python samples contain [requirements.txt](https://github.com/openvinotoolkit/openvino/blob/master/samples/python/hello_classification/requirements.txt) files, which are required to be installed before sample run. It's not mentioned in documentation. Maybe we can add this information to "Build" section, which is currently empty for Python samples. 
image

**Comment by sgolebiewski-intel** [comment]
I have applied your suggestions with the https://github.com/openvinotoolkit/openvino/pull/21661/commits/3afeb52107a45c69b466906d80186a2fe657eb43 commit.

**Comment by ilya-lavrenov** [comment]
1. Is it possible to align step 3. indentation with others? BTW, it looks OK for macOS
image
2. It states that if model is not IR or ONNX. I suppose we don't need to mention it, because other formats are also supported. Maybe like in other samples we should recommend to convert model either via OVC or convert_model ? It's better to teach users to use IR
image
3. MO is still used in this sample
image
4. Do we need a note about conversion to IR and that sample also supports ONNX? I suppose we can drop it, because later we demonstrate conversion to IR anyway. The same comment for [Throughput Benchmark Sample](http://openvino-doc.iotg.sclab.intel.com/seba-test-4/openvino_sample_throughput_benchmark.html#)
image

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**